bower.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "sb-admin",
+  "version": "0.0.0",
+  "dependencies": {
+    "angular": "1.2.16",
+    "json3": "~3.3.1",
+    "es5-shim": "~3.1.0",
+    "angular-resource": "1.2.16",
+    "angular-cookies": "1.2.16",
+    "angular-sanitize": "1.2.16",
+    "angular-animate": "1.2.16",
+    "angular-touch": "1.2.16",
+    "angular-route": "1.2.16",
+    "font-awesome": "4.3.0",
+    "angular-bootstrap": "0.12.0",
+    "oclazyload": "~0.5.2",
+    "angular-loading-bar": "~0.7.0",
+    "angular-ui-router": "~0.2.13",
+    "angular-toggle-switch": "~1.2.1",
+    "metisMenu": "~1.1.3",
+    "angular-chart.js": "~0.5.2",
+    "d3": "^3.3.13",
+    "metisMenu":"1.1.3",
+    "nvd3": "^1.7.1",
+    "leaflet": "0.7.x",
+    "jquery": "1.9.1 - 2",
+    "angular-nvd3":"^1.0.7",
+    "angular-leaflet-directive": "^0.10.0",
+    "angular-cookies": "1.2.16"
+    
```

Comment:
I am missing angular material here. Is the template based on angular-bootstrap? Is it hard to transit to angular material? Let's decide whether to got for material or bootstrap. I am a little biased to material because I recently did two projects with it.